Trying to use the "Close any open GUI for" block causes build errors.

### Issue description

Trying to use the "Close any open GUI for" block causes build errors.
_This seems to already be a reported issue, but the person who made it did not follow the bug report template. This should be a better report._

### How to reproduce this issue?

Step 1: Create a procedure
Step 2: Add the "Close any open GUI for" block
Step 3: Save

### Operating system

Windows

### Details

MCreator 2022.3
Latest Dev Build. (Pulled github repo and zipped the "src" folder.)

### Example workspace

_This is insanely easy to replicate, so yea..._

### Logs

```
Executing Gradle task: build 
Build info: MCreator 2022.3.48217, fabric-1.19.2, 64-bit, 16332 MB, Windows 10, JVM 17.0.3, JAVA_HOME: C:\Users\Micasddsa\Documents\Micasddsa's Addons\MCreator20223\jdk, started on: 2022-11-28-18:32:20 
  
> Configure project : 
Fabric Loom: 1.0.12 
Not publishing sources jar as it was not found. Use java.withSourcesJar() to fix. 
> Task :compileJava FAILED 
[C:\Users\Micasddsa\MCreatorWorkspaces\guitest\src\main\java\net\mcreator\guitest\procedures\GuiCloseProcedure.java:20](file:///C:/Users/Micasddsa/MCreatorWorkspaces/guitest/src/main/java/net/mcreator/guitest/procedures/GuiCloseProcedure.java:20): error: closeContainer() has protected access in Player 
         _player.closeContainer(); 
                ^ 
1 error 
FAILURE: Build failed with an exception. 
* What went wrong: 
Execution failed for task ':compileJava'. 
> Compilation failed; see the compiler error output for details. 
* Try: 
> Run with --stacktrace option to get the stack trace. 
> Run with --info or --debug option to get more log output. 
> Run with --scan to get full insights. 
* Get more help at https://help.gradle.org 
BUILD FAILED in 2s 
1 actionable task: 1 executed
```

### Issue tracker rule checks (please read carefully)

- [X] I have checked that my problem <a href='https://github.com/MCreator/MCreator/issues?q=is%3Aissue' target='_blank'>is not already reported</a>
- [X] I have checked that my problem is not covered on <a href='https://mcreator.net/support/knowledgebase' target='_blank'>Knowledge Base</a> or on <a href='https://mcreator.net/wiki' target='_blank'>MCreator's Wiki</a>
- [X] I have checked that my written content does not violate the <a href='https://mcreator.net/wiki/general-publishing-guidelines' target='_blank'>publishing guidelines</a>

--- a/changelog.md
+++ b/changelog.md
@@ -1,6 +1,7 @@
 # Changelogs:
 
 ## 1.7.1
+* [Bugfix #307] "Close any open GUI for" block caused a build error
 * [Bugfix #313] Mods could not be exported (once again)
 
 ## 1.7

--- a/src/fabric-1.19.2/procedures/entity_close_gui.java.ftl
+++ b/src/fabric-1.19.2/procedures/entity_close_gui.java.ftl
@@ -1,1 +1,1 @@
-if(${input$entity} instanceof Player _player) _player.closeContainer();
+if(${input$entity} instanceof ServerPlayer _player) _player.closeContainer();

